src/libcollections/vec.rs

```diff
@@ -528,7 +528,7 @@ impl<T> Vec<T> {
     }
 
     /// Inserts an element at position `index` within the vector, shifting all
-    /// elements after position `i` one position to the right.
+    /// elements after the position to the right.
```

Comment:
I found this sentence hard to parse, not realizing "the postion" was referencing that from the previous sentence. What about "shifting all elements after *it* to the right"?